ui/src/components/summaryitemcomponents/ArtillerySummary.svelte

```diff
@@ -1,24 +1,28 @@
 <script>
   export let value = {};
+
+  function numberWithCommas(x) {
+    return Math.round(x).toLocaleString()
+  }
 </script>
 
-<div class="grid grid-cols-2 lg:grid-cols-4">
+<div class="grid grid-cols-2 lg:grid-cols-4 gap-x-5">
   <div class="col-span-1 text-start">
     <span class="block font-sans">Latency P95</span>
     <span
-      class="font-sans font-bold block lg:inline-block">{value.latencyP95 === undefined ? '0' : value.latencyP95}
+      class="font-sans font-bold block lg:inline-block">{numberWithCommas(value.latencyP95 === undefined ? '0' : value.latencyP95)}ms
     </span>
   </div>
   <div class="col-span-1 text-start">
-    <span class="block whitespace-no-wrap font-sans">REQUESTS</span>
+    <span class="block whitespace-no-wrap font-sans">Requests</span>
     <span class="font-sans font-bold block lg:inline-block">
-      {value.requestsCompleted}
+      {numberWithCommas(value.requestsCompleted)}ms
```

Comment:
We don't need the "ms" here as the Requests is just the count of the number of requests that were sent.

Reply:
Done - removed 'ms' from Request

![image](https://user-images.githubusercontent.com/67776356/120970324-a0be6000-c7ae-11eb-912a-ce4135557dfd.png)